src/plugins/messenger/index.tsx

```diff
@@ -80,7 +80,9 @@ const messengerPlugin: MessagePluginFactory = ({ React, styled }) => {
                     // @ts-ignore
                     if (action.type === 'web_url' && action.url) {
                         // @ts-ignore
-                        window.open(action.url, '_blank');
+                        const target = action.target === "_self" ? "_self" : "_blank";
+                        // @ts-ignore
+                        window.open(action.url, target);
```

Comment:
We could simply pass `action.target`, right? 
```suggestion
                        window.open(action.url, action.target);
```

Reply:
Target needs to be "_self" or "_blank" otherwise it will run into errors. I tried it like this first, but then changed it to get sure it's always set to one of these values. This should be the best approach as previous versions of Cognigy AI didn't send a target property and would therefore become incompatible.

Reply:
Ah, now I get it...yes sure, I can modify it so we don't need another variable :)